go/libkb/chain_link_v2_test.go

```diff
@@ -110,3 +111,51 @@ func TestOuterLinkV2WithMetadataPointerContainerDecode(t *testing.T) {
 	err = MsgpackDecode(&o, bytes)
 	requireErrorHasSuffix(t, errCodecDecodeSelf, err)
 }
+
+func serdeOuterLink(t *testing.T, hPrevSeqno *keybase1.Seqno, hPrevHash LinkID, addHPrevHash bool) OuterLinkV2 {
+	o := OuterLinkV2{Version: 2, Seqno: keybase1.Seqno(2), Prev: nil, Curr: nil, LinkType: 1, SeqType: 1, IgnoreIfUnsupported: false}
```

Comment:
can you swap out these integers with the named constants, i.e. `LinkType: 1` with `LinkType: SigchainV2TypeEldest` and `SeqType: 1` with `SeqType: keybase1.SeqType_PUBLIC`?